redash/models/__init__.py

```diff
@@ -791,14 +791,25 @@ def evaluate(self):
         data = json_loads(self.query_rel.latest_query_data.data)
 
         if data['rows'] and self.options['column'] in data['rows'][0]:
+            operators = {
+                '>': lambda v, t: v > t,
+                '>=': lambda v, t: v >= t,
+                '<': lambda v, t: v < t,
+                '<=': lambda v, t: v <= t,
+                '==': lambda v, t: v == t,
+                '!=': lambda v, t: v != t,
```

Comment:
WDYT about replacing that "long" operator names with these? (old ones kept for backward compatibility: already created alerts can use them, but new ones will use new operators)

Reply:
I'd also consider the dedicated chars like `≥`.

Reply:
User does not see them; in dropdown we can show whatever we want, but here I'd prefer to use ANSI characters like we do for variable names.

Reply:
@ranbena why? I don't even know how to type those without looking up some unicode characters map.

Reply:
> use ANSI characters like we do for variable names.

Cool.

> why?

Cleaner when implementing on the Select component.

Reply:
> Cleaner when implementing on the Select component.

Not sure how it will help. Imagine that we'll add some new operation that will not have a dedicated Unicode symbol, e.g. `contains` or `match regexp`. IMHO these identifiers are for internal use and shouldn't be used for dropdown option names.

Reply:
I agree. I meant sth else. Doesn't matter. Let's land it.